internal/users/db/update.go

```diff
@@ -85,7 +85,7 @@ func handleGroupsUpdate(db queryable, groups []GroupRow) error {
 			return fmt.Errorf("GID for group %q already in use by a different group", group.Name)
 		}
 
-		log.Debug(context.Background(), fmt.Sprintf("Updating entry of group %q (%+v)", group.Name, group))
+		log.Debugf(context.Background(), "Updating entry of group %q (%+v)", group.Name, group)
```

Comment:
nitpick: this (together with the change in line 67) should be a separate commit

Reply:
oh they were!

Maybe I squashed them wrongly? Thanks for noticing it!